content/docs/gke/authentication.md

```diff
@@ -0,0 +1,181 @@
++++
+title = "Authenticating Kubeflow to GCP"
+description = "Authentication and authorization to GCP"
+weight = 4
++++
+
+
+# Authenticating gcloud
+
+[The `gcloud` tool](https://cloud.google.com/sdk/gcloud/) is used to interact with Google Cloud Platform (GCP) over the command line. 
+You can use the `gcloud` command to [set up Google Kubernetes Engine (GKE) clusters](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create), 
+and interact with other Google services.
+
+### Logging in
+
+You have two options for authenticating the `gcloud` command:
+
+
+- You can create a **service account** within your GCP project. You can then
+[download a `.json` key file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) 
+associated with the account, and run the 
+[`gcloud auth activate-service-account`](https://cloud.google.com/sdk/gcloud/reference/auth/activate-service-account)
+command to authenticate your `gcloud` session.
+This is the recommended way to authenticate with GCP.
+
+- You can use a **user account** to authenticate using a Google account (typically Gmail). 
+You can register a user account using [`gcloud auth login`](https://cloud.google.com/sdk/gcloud/reference/auth/login), 
+which brings up a browser window to start the familiar Google authentication flow.
+
+You can find more information in the [GCP docs](https://cloud.google.com/sdk/docs/authorizing).
+
+### Listing active accounts
+
+You can run the following command to verify you are authenticating with the expected account. 
+In the output of the command, an asterisk denotes your active account.
+
+```
+gcloud auth list
+```
+
+### Viewing IAM roles
+
+Permissions are handled in GCP using [IAM Roles](https://cloud.google.com/iam/docs/understanding-roles). 
+These roles define which resources your account can read or write to. You can check which roles were assigned to your account using the following gcloud command:
+
+```
+PROJECT_ID=your-gcp-project-id-here
+
+gcloud projects get-iam-policy $PROJECT_ID --flatten="bindings[].members" \
```

Comment:
I think this will only work if the user has IAM viewer permissions. So someone who has access to a Kubeflow cluster might not have this permission.